src/pages/List.js

```diff
@@ -5,18 +5,47 @@ export default function List(props) {
   const [listItem, loading, error] = useCollection(db.collection(props.token), {
     snapshotListenOptions: { includeMetadataChanges: true },
   });
+
+  const markItemPurchased = (e, id) => {
+    const elapsedMilliseconds = Date.now();
+
+    if (e.target.checked === true) {
+      db.collection(props.token).doc(id).update({
+        last_purchased: elapsedMilliseconds,
+      });
+    } else {
+      db.collection(props.token).doc(id).update({
+        last_purchased: null,
+      });
+    }
+  };
+
+  function compareTimeStamps(lastPurchased) {
+    const currentElapsedMilliseconds = Date.now();
+    const millisecondsInOneDay = 86400000;
```

Comment:
I like your choice in naming this variable. It clearly self-documents what the number is.